_layouts/meta.html

```diff
@@ -7,13 +7,16 @@
     <meta name="description" content="Hackerspace to społeczność i miejsce dla ludzi, którym po prostu się chce!">
     
     <link rel="manifest" href="/manifest.json">
-    <link rel="icon" type="image/png" sizes="32x32" href="/favicon/hsp-32x32.png">
-    <link rel="icon" type="image/png" sizes="16x16" href="/favicon/pixel-hsp.png">
-    <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
-    <link rel="mask-icon" href="/favicon/safari-pinned-tab.svg" color="#737373">
 
-    <meta name="msapplication-TileColor" content="#2b5797">
-    <meta name="theme-color" content="#000000">
+    <link rel="apple-touch-icon" sizes="180x180" href="/favicon/apple-touch-icon.png">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon/favicon-32x32.png">
+    <link rel="icon" type="image/png" sizes="16x16" href="/favicon/favicon-16x16.png">
+    <link rel="manifest" href="/favicon/site.webmanifest">
```

Comment:
Now we have two manifest files. Remove one and edit the other to reflect the changes.